ol/cli/src/cache.rs

```diff
@@ -63,8 +62,8 @@ impl Vitals {
 
         // after writing temporary file renames and overwrite to cache file
         let cache_path = get_cache_path(node_home);
-        rename(temp_path, cache_path).expect("temporary cache file should be renamed");
-        Ok(())
+        let result = rename(temp_path, cache_path)?;
```

Comment:
You can simply end the block of code with 

`rename(temp_path, cache_path)`

No need for the assignment and then the Ok()